src/ManageCourses.Api/McConfig.cs

```diff
@@ -25,6 +31,13 @@ public void Validate()
             ValidateRequired(PgServerKey);
             ValidateRequired(PgDatabaseKey);
             ValidateRequired(PgUsernameKey);
+            ValidateRequired(SignInUserInfoEndpointConfigKey);
+            ValidateRequired(ApiKeyConfigKey);
+            //ValidateRequired(EmailApiKeyConfigKey);
```

Comment:
Add comments to denote that these are optional values to determine the instance of the email client generated please